ngraph/frontend/generic/include/frontend_manager/frontend_manager_defs.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/visibility.hpp"
+
+#define OV_FRONTEND_API_VERSION "0.0.1"
```

Comment:
can it be a numeric values? In this case we can use conditional compilation in 3rd party apps

Reply:
Ok, changed to 1